packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1394,6 +1394,7 @@ export default class Meeting extends StatelessWebexPlugin {
       );
 
       if (this.mediaProperties.remoteStream.getVideoTracks()[1]) {
+        this.mediaProperties.setRemoteShare(MediaUtil.createMediaStream([this.mediaProperties.remoteStream.getVideoTracks()[1]]));
```

Comment:
To improve the readability of the changes in this line, could we do the following:

```js
this.mediaProperties.setRemoteShare(
  MediaUtil.createMediaStream([
    this.mediaProperties.remoteStream.getVideoTracks()[1]
  ])
);
```